crates/db_schema/src/impls/actor_language.rs

```diff
@@ -329,17 +329,21 @@ mod tests {
   use serial_test::serial;
 
   async fn test_langs1(pool: &DbPool) -> Vec<LanguageId> {
-    vec![
+    let mut vec = vec![
       Language::read_id_from_code(pool, "en").await.unwrap(),
       Language::read_id_from_code(pool, "fr").await.unwrap(),
       Language::read_id_from_code(pool, "ru").await.unwrap(),
-    ]
+    ];
+    vec.sort();
+    vec
```

Comment:
These are the static ones, why do they need to be sorted? Isn't it just the queries that should be sorted predictably?